dev/tests/integration/testsuite/Magento/Customer/_files/three_customers_rollback.php

```diff
@@ -0,0 +1,43 @@
+<?php
```

Comment:
Please looks like I do it for `two_customers_rollback.php`

Reply:
@naydav , I do not like `deleteById` solution here:
```php
$customerRepository->deleteById(2);
```
I will explain why.

If you take a look to `Magento\GraphQl\Quote\Customer\GetAvailablePaymentMethodsTest`::`testGetPaymentMethodsFromAnotherCustomerCart()` you will see that for this test we use the next fixtures:
- Magento/Checkout/_files/quote_with_items_saved.php
- Magento/Customer/_files/three_customers.php

Fixture `quote_with_items_saved.php` uses `Customer/_files/customer.php` which creates a customer with `entity_id = 1` and `email = customer@example.com`.

In this case `three_customers.php` is unable to create `customer@search.example.com` with `entity_id=1` because customer with such entity_id is already exist. 

Please see: https://github.com/magento/graphql-ce/blob/2.3-develop/dev/tests/integration/testsuite/Magento/Customer/_files/three_customers.php

**My opinion:** we should set entity_id for the customer fixtures. If you agree then this is a stuff for the new ticket.

Please review and provide your suggestions. 

Thank you!

Reply:
@atwixfirster 
You can't use 
`Magento/Customer/_files/customers.php`
`Magento/Customer/_files/three_customers.php`
at the same time because both of fixture contains hardcoded `customer_id` thus this fix will not resolve the descibed problem.

BUT, you should not use both of these fixtures at the same time. They are designed for different scenarios.

Reply:
Understood , @naydav 👍 

I've made a rollback for changes in the `Magento/Customer/_files/customers.php` and left `Magento/Customer/_files/three_customers.php` only.

Thank you again for the explanations!

Reply:
@naydav a quick notice for you.

It's not a good point to use both fixtures for the same test:
- Magento/Customer/_files/customers.php
- Magento/Customer/_files/three_customers.php

However , when we use 
- Magento/Customer/_files/three_customers.php
- Magento/Checkout/_files/quote_with_items_saved.php

then `quote_with_items_saved.php` includes `customers.php`.